tests/BlazorServerCallsGraph/Pages/CallWebApi.razor

```diff
@@ -20,13 +20,25 @@ else
 }
 
 @code {
-    private string apiResult;
+    private HttpResponseMessage response;
+    string apiResult;
 
     protected override async Task OnInitializedAsync()
     {
         try
         {
-            apiResult = await downstreamAPI.CallWebApiAsync("me");
+            response = await downstreamAPI.CallWebApiForUserAsync(
+                    "CalledApi",
+                    options => options.RelativePath = "me");
+
+            if (response.StatusCode == System.Net.HttpStatusCode.OK)
+            {
+                apiResult = await response.Content.ReadAsStringAsync();
+            }
```

Comment:
Here we use the pattern recommended by Damien, but we could also use the strongly typed wrapper CallWebApiForUserAsync<InputType, ReturnType> which makes it even easier. It's used in the TodoListService wrapper in the TodoListClient project.